tests/unit/test_utils.py

```diff
@@ -78,6 +78,7 @@ def mock_unpack(link, source_dir, download_dir, only_download=False, session=Non
 
 @pytest.mark.utils
 @pytest.mark.parametrize("deps, expected", DEP_PIP_PAIRS)
+@pytest.mark.network
```

Comment:
We already have a predefined marker `needs_internet` which can automatically determine whether to run particular cases depending on the network condition.

Reply:
Ah, ok, I'll change it to that.

Reply:
So, when changed to `@pytest.mark.needs_internet` the tests are not automatically skipped when there's no internet connection:

    FAILED tests/unit/test_utils.py::TestUtils::test_download_file - requests.exc...
    FAILED tests/unit/test_utils.py::test_convert_deps_to_pip[deps10-https://github.com/Rapptz/discord.py/archive/async.zip#egg=discord.py[voice]]
    =============== 2 failed, 97 passed, 9 skipped in 13.98 seconds ================

However, I can just skip them manually, so it doesn't really matter to this PR.

Reply:
@torsava that will be true if you aren't invoking the entire test suite, since the relevant configuration happens in `tests/integration/conftest.py` (could use some refactoring but nobody has gotten around to it)

Reply:
Ah, makes sense. Yeah, I can't invoke the whole test suite, because it fails while gathering test cases si if there's no internet.